libs/wingc/src/lsp/hover.rs

```diff
@@ -169,13 +169,19 @@ impl<'a> Visit<'a> for HoverVisitor<'a> {
 				value,
 				statements,
 				reassignable: _,
+				elif_statements,
 				else_statements,
 			} => {
 				self.with_scope(statements, |v| {
 					v.visit_symbol(var_name);
 				});
 				self.visit_expr(value);
 				self.visit_scope(statements);
+				for elif in elif_statements {
+					self.visit_symbol(&elif.var_name);
+					self.visit_expr(&elif.value);
+					self.visit_scope(&elif.statements);
+				}
```

Comment:
Weird thing to handle with `if let`-stuff: Technically the symbol is defined within the inner scope, so we have to let the visitor know

```suggestion
				for elif in elif_statements {
				  self.with_scope(&elif.statements, |v| {
				  	v.visit_symbol(&elif.var_name);
				  });
					self.visit_expr(&elif.value);
					self.visit_scope(&elif.statements);
				}
```